### ISSUE TITLE
badIndex in Native/Json.js defined twice

### ISSUE THREAD

**Issue by mzero** [elm-lang/core#936]
The function `badIndex` is defined twice in the file Native/Json.js:

[line 166](https://github.com/elm-lang/core/blob/5.1.1/src/Native/Json.js#L166)
[line 176](https://github.com/elm-lang/core/blob/5.1.1/src/Native/Json.js#L176)

The definitions are identical.
This isn't a problem for JavaScript, but causes Closure Compiler to barf:

```
JSC_BLOCK_SCOPED_DECL_MULTIPLY_DECLARED_ERROR: Duplicate let / const / class / function declaration in the same scope is not allowed. at line 6664 character 9
function badIndex(index, nestedProblems)
```

**Comment by zwilias** [comment]
This got fixed in the `dev` branch so will be included in the next release of `elm-lang/core`. Thanks for logging the issue!

**Comment by YetAnotherMinion** [comment]
It is cool that this bug has been fixed for Elm 0.19, but there are a lot of people still using Elm 0.18 because **0.19 has not been released yet**! Is there a process for backporting the fix to the 5.1 version of this library (I will do all the work)? Looking at the git history, it does not look like release branches are being used, is there opposition to creating one from the 5.1.1 tag? I could then cherry pick the badIndex fix onto it, and a member could release a 5.1.2 bugfix for all Elm users in ~5 minutes.
